tsconfig.json

```diff
@@ -22,7 +22,7 @@
     "downlevelIteration": true /* Provide full support for iterables in 'for-of', spread, and destructuring when targeting 'ES5' or 'ES3'. */,
     // "isolatedModules": true,               /* Transpile each file as a separate module (similar to 'ts.transpileModule'). */
     /* Strict Type-Checking Options */
-    "strict": true /* Enable all strict type-checking options. */,
+    "strict": false /* Enable all strict type-checking options. */,
```

Comment:
I think you can add the following lines to `src/@types/relay-compiler.d.ts` and revert this file.

```ts
declare module "relay-compiler/lib/transforms/RefetchableFragmentTransform";
declare module "relay-compiler/lib/transforms/MaskTransform";
declare module "relay-compiler/lib/transforms/MatchTransform";
declare module "relay-compiler/lib/transforms/RelayDirectiveTransform";
```

Reply:
If only it was that simple. With TS in strict mode, you'd get:

> Invalid module name in augmentation. Module 'relay-compiler/lib/transforms/RefetchableFragmentTransform' resolves to an untyped module at 'c:/code/relay-compiler-language-typescript/node_modules/relay-compiler/lib/transforms/RefetchableFragmentTransform.js', which cannot be augmented.

I guess we'll just have to wait for https://github.com/DefinitelyTyped/DefinitelyTyped/pull/39665 to be merged.